### PR TITLE
Add vetting, and test coverage for drupal settings

### PR DESCRIPTION
**Pull request by simesy**

### CODE REVIEW

**Comment by simesy** [comment]
@todo Check the drush/drush.yml is default.

**Comment by simesy** [comment]
closes issues/15

**Comment by simesy** [comment]
closes govCMS/scaffold-tooling#15

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,9 +2,10 @@ version: 2
 jobs:
   build:
     docker:
-      - image: integratedexperts/ci-builder
+      - image: quay.io/govcms/govcms-ci:shellcheck_bats
```
Comment (tobybellwood) [inline .circleci/config.yml]: (now merged into master)

**Review by steveworley — APPROVED** [review]

**Comment by simesy** [comment]
Thanks @steveworley 🎉

**Pull request merged**